### PR TITLE
chore: remove unused babel plugins

### PR DESCRIPTION
**Pull request by HHongSeungWoo**
## Description

removed the unused Babel plugin

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a storybook story or extended an existing story to show my changes
- [x] All new and existing tests passed.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -19,12 +16,6 @@
     "@babel/preset-typescript"
```
Comment (nikolasrieble) [inline .babelrc:16]: This should then also go, right?

File: .babelrc [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {
   "plugins": [
-    "@babel/plugin-proposal-export-default-from",
-    "@babel/plugin-proposal-function-bind",
-    "babel-plugin-lodash",
     "babel-plugin-dev-expression"
```
Comment (nikolasrieble) [inline .babelrc:3]: Are the other plugins still used?

**Review by nikolasrieble — APPROVED** [review]
Thank you for the cleanup.

**Review by ckifer — APPROVED** [review]
Why are these all unused? because we dropped support for ES5?

**Pull request merged**